File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-1.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int length = nums.length, l = 1, r = 1;
        while(r<length){
            if(nums[r] == nums[r-1]){
                r++;
            }else {
                nums[l] = nums[r];
                r++;
                l++;
            }
        }
        return l;
    }
}

File: Data Structures & Algorithms/minimum-difference-between-highest-and-lowest-of-k-scores/submission-1.py
class Solution:
    def minimumDifference(self, nums: List[int], k: int) -> int:
        nums.sort()
        n = len(nums)
        min_diff = nums[n-1]

        for i in range(0, n - k + 1):
            min_diff = min(min_diff, nums[i + k - 1] - nums[i])
        return min_diff